rllib/core/examples/simple_ppo_rl_module.py

```diff
@@ -210,110 +216,166 @@ def input_specs_inference(self) -> ModelSpec:
 
     @override(RLModule)
     def output_specs_inference(self) -> ModelSpec:
-        return ModelSpec(
-            {
-                "action_dist": TorchDeterministic,
-            }
-        )
+        return ModelSpec({SampleBatch.ACTION_DIST: TorchDeterministic})
```

Comment:
Dumb question: Shouldn't this be `TorchCategorical`?

Reply:
Ah, got it: `output_specs_exploration` vs `output_specs_inference`.
Never mind.

Reply:
yep.